packages/babel-cli/test/fixtures/babel/env - SHOW_CONFIG_FOR/in-files/my-extended.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
             "@foo/babel-preset-1",
             [
               {
-                name: "@foo/inline-babel-preset-1",
```

Comment:
Q: What is this change needed for?

Reply:
Babel will throw `name` is not a valid option. I didn't know `name` is forbidden on `preset` until now. 🤦‍♂️